src/Playwright.Xunit/WorkerAwareTest.cs

```diff
@@ -34,10 +34,10 @@
 
 namespace Microsoft.Playwright.Xunit;
 
-public class WorkerAwareTest : ExceptionCapturer, IAsyncLifetime
+public class WorkerAwareTest : ExceptionCapturer
 {
     private static readonly ConcurrentStack<Worker> _allWorkers = new();
-    private Worker? _currentWorker = null!;
+    private Worker _currentWorker = null!;
```

Comment:
drive-by: Similar to https://github.com/microsoft/playwright-dotnet/commit/7f3b8884efc0f288f504d90a677afbd3187f4fbe.